src/components/Comment/CommentOptionButton.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+import Button from "../common/Button";
+import CommentOptionDropdown from "./CommentOptionDropdown";
+
+import { ICON } from "../../constants/variants";
+
+const CommentOptionWrapper = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+`;
+
+export default function CommentOptionButton({ commentId, snippetId, userId, updateCommentList, setIsEditable }) {
+  const [isClicked, setIsClicked] = useState(false);
+
+  const UserMenuIcon = <img alt="댓글란 더보기 아이콘" src="/images/more.png" width="40" height="40" />;
+
+  const handleDropDown = () => {
+    setIsClicked(!isClicked);
+  };
+
+  return (
+    <CommentOptionWrapper>
+      <Button variant={ICON} onClick={handleDropDown} children={UserMenuIcon} />
+      {isClicked && <CommentOptionDropdown commentId={commentId} userId={userId} snippetId={snippetId} updateCommentList={updateCommentList} setIsEditable={setIsEditable} />}
```

Comment:
```suggestion
      {isClicked && (
        <CommentOptionDropdown 
          commentId={commentId}
          userId={userId}
          snippetId={snippetId}
          updateCommentList={updateCommentList}
          setIsEditable={setIsEditable} 
        />)}
```
가독성을 위해서 개행해 주시면 좋을 것 같습니다

Reply:
props를 다섯 개 이상 가지거나 그 이하여도 props 네임이 길어 가독성을 해치는 경우 코멘트 주신 것과 같이 수정하겠습니다